packages/core/src/App/Constants/routes-config.js

```diff
@@ -80,10 +87,119 @@ const getModules = ({ is_deriv_crypto }) => {
+            icon_component: 'IcWalletPlatforms',
+            getTitle: () => localize('Platforms'),
+            routes: [
+                {
+                    default: true,
+                    getTitle: () => localize('DMT5'),
                     path: routes.platform_dmt5_synthetic,
-                    component: Dashboard,
-                    getTitle: () => localize('DMT5 Synthetic'),
+                },
+                {
+                    getTitle: () => localize('DTrader'),
+                    // path: routes.platform_dtrader,
+                    path: routes.explore,
+                },
+                {
+                    getTitle: () => localize('DBot'),
+                    // path: routes.platform_dbot,
+                    path: routes.explore,
+                },
+                {
+                    getTitle: () => localize('SmartTrader'),
+                    // path: routes.platform_smarttrader,
+                    path: routes.explore,
+                },
+                {
+                    getTitle: () => localize('Binary Bot'),
+                    // path: routes.platform_binarybot,
+                    path: routes.explore,
+                },
+            ],
+        },
+        {
+            path: routes.trade_types,
+            icon_component: 'IcWalletTradeTypes',
+            getTitle: () => localize('Trade Types'),
+            routes: [
+                {
+                    getTitle: () => localize('CFDs'),
+                    // path: routes.trade_types_cfds,
+                    path: routes.explore,
+                },
+                {
+                    getTitle: () => localize('Multipliers'),
+                    // path: routes.trade_types_multipliers,
+                    path: routes.explore,
+                },
+                {
+                    getTitle: () => localize('Options'),
+                    // path: routes.trade_types_options,
+                    path: routes.explore,
+                },
+            ],
+        },
+        {
+            path: routes.markets,
+            icon_component: 'IcWalletMarkets',
+            getTitle: () => localize('Markets'),
+            routes: [
+                {
+                    getTitle: () => localize('Forex'),
+                    // path: routes.markets_forex,
+                    path: routes.explore,
+                },
+                {
+                    getTitle: () => localize('Synthetic Indices'),
+                    // path: routes.markets_synthetic,
+                    path: routes.explore,
+                },
+                {
+                    getTitle: () => localize('Stock Indices'),
+                    // path: routes.markets_stock,
+                    path: routes.explore,
+                },
+                {
+                    getTitle: () => localize('Commodities'),
+                    // path: routes.markets_commodities,
+                    path: routes.explore,
```

Comment:
why paths are commented?

Reply:
These pages are not ready yet, so I just route to Explore instead. We can uncomment it once the static pages are done